### PR TITLE
New User section in Kaui Guide plus changes due to adding 2 glossary terms 

### PR DESCRIPTION
**Pull request by ghost**
I've finally finished reworking this. There are a few new things to note: 

- I added the terms "User role" and "Permissions" to the Glossary, so you'll see that file has been affected.

- I also added these terms to the mini-glossary at the beginning of the main Kaui Guide (I plan to break this guide into smaller files at some point.)

- I created a section, "Open the User List." This lets me just reference it in the edit and delete sections instead of repeating steps across sections.

- I've clarified the difference between deleting a user from the system and removing a user from the tenant. (I know this isn't quite settled, but I just wrote it the way I think it should work. <grin> I can always rewrite it after Reshma tests.)

- In the field description table, I have a question about how the "User name" should be formatted. Is it a first name, last name, both, or an email, or is this completely up to the organization using Kill Bill? Does it support spaces? I can't test this stuff, you know.

### CODE REVIEW

File: userguide/kaui/includes/users.adoc [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+. <<Open the User List,Open the user list.>>
++
+image::UsersList3.png[width=650,align="center"]
++
+. Select the user you want to edit.
++
+Kaui displays the User Details screen.
++
+image::UserDetails.png[width=450,align="center"]
++
```
Comment (reshmabidikar) [inline userguide/kaui/includes/users.adoc:112]: When a user name is clicked in the `Users` screen, this displays the attached screen:

edit-user

You then need to click the "Edit" link highlighted above to view the "Update User" screen.

File: userguide/kaui/includes/users.adoc [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+. <<Open the User List,Open the user list.>>
++
+image::UsersList3.png[width=650,align="center"]
++
+. Select the user you want to delete.
++
+Kaui displays the User Details screen.
++
+image::UserDetails.png[width=450,align="center"]
++
```
Comment (reshmabidikar) [inline userguide/kaui/includes/users.adoc:162]: Same comment as above, when a user name is clicked in the `Users` screen, this displays the screen attached above.

File: userguide/kaui/includes/users.adoc [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+. Click the image:i_RedX.png[] icon. Kaui displays a message to confirm the deletion.
++
+. Click *OK*.
+
```
Comment (reshmabidikar) [inline userguide/kaui/includes/users.adoc:193]: After clicking `OK`, the Kaui login screen is displayed and a user is asked to log in again. After successfully logging in, the user is actually deleted.

File: userguide/kaui/includes/users.adoc [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
++
+image::AllowedUsersList.png[width=650,align="center"]
++
+Kaui displays the User Details screen.
++
+image::UserDetails.png[width=450,align="center"]
++
+. Click the image:i_RedX.png[] icon. Kaui displays a message to confirm the deletion.
++
+. Click *OK*.
```
Comment (reshmabidikar) [inline userguide/kaui/includes/users.adoc:211]: The behavior I see is slightly different, please see the attached screenshot:
tenant-screen
Let me know if this isn't clear or if you need more information

Comment (reshmabidikar) [reply userguide/kaui/includes/users.adoc:211]: Secondly, I think you missed documenting `Add Allowed User`. In the tenant screen, you can add a user (allowed to access the tenant), by clicking the `+` sign next to `Allowed Users`

**Review by reshmabidikar — COMMENTED** [review]
Hi Mary, looks really good. I found some differences in edit/delete users' functionality between what you've documented and what I see in my local installation. I've added comments to indicate this. I've also created a short screen recording to show how this works on my local installation, attaching it here. (I had to compress it since the file was large). 
[kaui-clip.zip](https://github.com/killbill/killbill-docs/files/8048641/kaui-clip.zip)
 Let me know if you need further information.

**Comment by sbrossie** [comment]
Looking great - @FlowWriterAny objection if I merge this PR, and you can open a new one with the couple of comments raised by  @reshmabidikar ?

File: userguide/kaui/includes/users.adoc (hunk 3 above)
Comment (ghost) [reply userguide/kaui/includes/users.adoc:193]: > Looking great - @FlowWriterAny objection if I merge this PR, and you can open a new one with the couple of comments raised by @reshmabidikar ?

I'm right in the middle of editing; not sure if that will cause problems or not.

File: userguide/kaui/includes/users.adoc (hunk 1 above)
Comment (ghost) [reply userguide/kaui/includes/users.adoc:112]: Fixed

File: userguide/kaui/includes/users.adoc (hunk 2 above)
Comment (ghost) [reply userguide/kaui/includes/users.adoc:162]: Fixed

File: userguide/kaui/includes/users.adoc (hunk 3 above)
Comment (ghost) [reply userguide/kaui/includes/users.adoc:193]: Fixed

File: userguide/kaui/includes/users.adoc (hunk 4 above)
Comment (ghost) [reply userguide/kaui/includes/users.adoc:211]: Fixed - also added Add an Allowed User section.

**Comment by ghost** [comment]
Added a step in Edit the User to change the tenant with the checkboxes - I hope that's correct. 

Can you let me know about the Name field in the Fields Description table?

In Step 3 of Remove a User from the Tenant, I have them clicking "OK," but I'm not sure if there's a confirmation message or not.

**Comment by ghost** [comment]
I regenerated the PDF.

**Review by reshmabidikar — APPROVED** [review]
In `Remove a User from the Tenant`, after Step 2 (click the black x), the tenant is deleted. So, there is no step 3. Other than that, everything looks good. I've approved the changes.

**Comment by pierre** [comment]
👏 

Merging already since it's a lot of changes 😅 but see @reshmabidikar's comment above.

**Comment by sbrossie** [comment]
👏

**Pull request merged**